apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -42,10 +42,14 @@ export default function LearningGoal({
   setFeedbackAdded,
   aiEvalInfo,
 }) {
+  const STATUS = Object.freeze({
```

Comment:
Since this is duplicated in two components it might be nice to put it in a shared location (perhaps `./rubricHelpers`?) and reference it from there.

Reply:
LearningGoal will be deprecated and replaced with LearningGoals once the full set of UI updates are live. I don't think these constants will be used in any other components.

Reply:
Fair enough. In that case keeping the declaration close to the point of use seems better for readability.